### PR TITLE
[WIP] Add setters

### PR DESCRIPTION
**Pull request by iamonuwa**
@Daniel-K-Ivanov @Perseverance please review.

Am I in order here?

1. Add setters to etherlime-ganache-deployer.js

### CODE REVIEW

File: deployer/etherlime-ganache-deployer/etherlime-ganache-deployer.js [hunk 1]
```diff
@@ -24,6 +24,18 @@ class EtherlimeGanacheDeployer extends JSONRPCDeployer {
 		this.nodeUrl = nodeUrl;
 	}
 
+	set privateKey(privateKey) {
```
Comment (Perseverance) [inline deployer/etherlime-ganache-deployer/etherlime-ganache-deployer.js]: No need to make this setter. Instead we are looking for fully fledged method like `setPrivateKey`

File: deployer/etherlime-ganache-deployer/etherlime-ganache-deployer.js [hunk 2]
```diff
@@ -14,7 +14,7 @@ class EtherlimeGanacheDeployer extends JSONRPCDeployer {
 	 * @param {*} port port number of the network to deploy on. This is the port number that is given to the class
 	 * @param {*} defaultOverrides [Optional] default deployment overrides
 	 */
-	constructor(privateKey = ganacheSetupConfig.accounts[0].secretKey, port = ganacheSetupConfig.defaultPort, defaultOverrides) {
+	constructor(privateKey = this.privateKey, port = this.port, defaultOverrides = this.defaultOverrides) {
```
Comment (Perseverance) [inline deployer/etherlime-ganache-deployer/etherlime-ganache-deployer.js]: This change is incorrect

File: deployer/etherlime-ganache-deployer/etherlime-ganache-deployer.js (hunk 1 above)
Comment (iamonuwa) [reply deployer/etherlime-ganache-deployer/etherlime-ganache-deployer.js]: Oh! I get that now.

File: deployer/etherlime-ganache-deployer/etherlime-ganache-deployer.js (hunk 2 above)
Comment (iamonuwa) [reply deployer/etherlime-ganache-deployer/etherlime-ganache-deployer.js]: I'll submit the fix soon

**Comment by Perseverance** [comment]
I think you've misunderstood the task at hand. We are looking to augment the Deployer (not only the Etherlime Deployer, but the base class too) with methods like `setPrivateKey`. We are not looking for setters `set port`. Also do please keep the standard default values of the parameters.

**Comment by Perseverance** [comment]
Closing due to inactivity

**Pull request closed without merging**